cmake/TestImpactFramework/LYTestImpactFramework.cmake

```
@@ -324,31 +324,64 @@ function(ly_test_impact_write_gem_target_enumeration_file GEM_TARGET_TEMPLATE_FI
      configure_file(${GEM_TARGET_TEMPLATE_FILE} ${mapping_path})
 endfunction()
 
+#! ly_extract_aliased_target_dependencies: recursively extracts the aliases of a target to retrieve the true de-aliased target.
+#
+# \arg:TARGET target to de-alias
+function(ly_extract_aliased_target_dependencies TARGET)
+    if(ARGN STREQUAL "")
```

This should check that ARGC is GREATER than 1 for validating whether additional parameters have been supplied.
```suggestion
    if(ARGC GREATER 1)
```

---

Fixed 👍